server/events/markdown_renderer.go

```diff
@@ -249,7 +248,8 @@ var planSuccessWrappedTmpl = template.Must(template.New("").Parse(
 // to do next.
 var planNextSteps = "{{ if .PlanWasDeleted }}This plan was not saved because one or more projects failed and automerge requires all plans pass.{{ else }}* :arrow_forward: To **apply** this plan, comment:\n" +
 	"    * `{{.ApplyCmd}}`\n" +
-	"* :put_litter_in_its_place: To **delete** this plan click [here]({{.LockURL}})\n" +
+	"* :put_litter_in_its_place: To **delete** this plan click [here]({{.LockURL}}), or to delete all plans and atlantis locks comment:\n" +
+	"    * `atlantis unlock`\n" +
```

Comment:
I'm going to move this to the bottom where we talk about running atlantis apply to apply all plans because unlocking all plans makes more sense there than nested with each individual plan

Reply:
Ah yea good point didn't think about it like that. I just kept it in the already existing `delete` section to avoid adding a new bullet point but yea sounds good 👍

Reply:
It's just gonna be a bit a of a pain to update the `rendere_tests` :-D